pkg/output/nameprinter.go

```diff
@@ -0,0 +1,54 @@
+package output
+
+import (
+	"io"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/cli-runtime/pkg/printers"
+)
+
+// PrintResourceNames prints the name of a resource, or if it's
+// a list, it will print the names of the nested resources.
+//
+// It can print deeply nested list objects names.
+func PrintResourceNames(out io.Writer, resource runtime.Object) error {
```

Comment:
This is necessary because `cli-runtime` utils don't support printing names for `List` types.